doc/changes/names.inc

```diff
@@ -363,3 +363,5 @@
 .. _Enrico Varano: https://github.com/enricovara/
 
 .. _Dominik Welke: https://github.com/dominikwelke/
+
+.. _Judy D Zhu: https://github.com/JD-Zhu
```

Comment:
@JD-Zhu can you add a line to the top of the `Bugs` section of `doc/changes/latest.inc` about this change? Follow the format for new contributors involving bolding, etc.

Reply:
In the description, should I reference the function that led to the error (mne.io.read_raw_kit), or the actual (internal?) function I modified (get_kit_info in mne/io/kit/kit.py)? I referenced the former - please let me know if that's incorrect. Thanks.

Reply:
We only refer to public-facing functions in `latest.inc`, so that's perfect!

(`get_kit_info` isn't really supposed to be public-facing or used by people even though it's not prefixed by an `_` like it probably should be.)